packages/@aws-cdk/aws-lambda-event-sources/lib/dynamodb.ts

```diff
@@ -17,6 +18,14 @@ export interface DynamoEventSourceProps {
    * Where to begin consuming the DynamoDB stream.
    */
   readonly startingPosition: lambda.StartingPosition;
+
+  /**
+   * The maximum amount of time to gather records before invoking the function.
+   * Maximum of Duration.minutes(5)
+   *
+   * @default Duration.seconds(0)
```

Comment:
I couldn't find documentation on the default, but I'm assuming that since this is a new feature, the previous behavior is the default.